web/src/main/webapp/swagger.yaml

```diff
@@ -770,6 +770,104 @@ paths:
               required:
                 - mission
                 - runtime
+   /osio/launch/empty:
+    post:
+      tags:
+        - Openshift.io
+      security:
+        - bearerAuth: []
+      parameters:
+        - name: X-App
+          in: header
+          description: >-
+            The Application where this request originated from (Openshift.io). This is necessary to determine how the Authorization
+            header is handled. If this header is not provided, the backend will
+            attempt to resolve this information from the Host header
+          required: true
+          schema:
+            type: string
+            enum:
+              - launcher
+              - osio
+            default: launcher
+        - name: X-Git-Provider
+          in: header
+          description: 'The Git provider to use. (GitHub, GitLab). Default is GitHub'
+          required: true
+          schema:
+            type: string
+            default: GitHub
+        - name: X-Execution-Step-Index
+          in: header
+          description: >-
+            The step index where the execution flow has stopped. Necessary when
+            requests must be retried
+          required: true
+          schema:
+            type: string
+            default: 0
+      summary: Launches the chosen booster
+      description: >-
+        Launches the empty booster (creates the github project, openshift
+        project and registers webhooks)
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Boom'
+        '400':
+          description: Validation Failed
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ValidationError'
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                mission:
```

Comment:
Are you still requiring mission in this case?

Reply:
yes, we require existing mission like `health-check` or `rest-http` as I discussed with you earlier.